includes/embeds/class-amp-soundcloud-embed.php

```diff
@@ -23,9 +30,8 @@ class AMP_SoundCloud_Embed_Handler extends AMP_Base_Embed_Handler {
 	 * Register embed.
 	 */
 	public function register_embed() {
-		if ( function_exists( 'soundcloud_shortcode' ) ) {
-			// @todo Move this to Jetpack.
-			add_shortcode( 'soundcloud', [ $this, 'shortcode' ] );
+		if ( ! $this->is_amp_shortcode_available_in_jetpack( self::SHORTCODE_TAG ) ) {
+			add_shortcode( self::SHORTCODE_TAG, [ $this, 'shortcode' ] );
```

Comment:
Please mark the `shortcode` methods as `@deprecated`.

Reply:
Sure, https://github.com/ampproject/amp-wp/pull/3678/commits/3e0e649706533cbe6fbf96cb5c63710f32887d93 does that.